### PR TITLE
Create flowchart-bedtime-routine.md

### PR DESCRIPTION
**Pull request by Yael0508**

### CODE REVIEW

File: student-work/2022/Yael-Lachman/flowchart-bedtime-routine.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+### Putting kids to bed
+```mermaid
+  graph TD;
+  A(Start)-->B[Eat supper]
```
Comment (Laura-Novich-OBW) [inline student-work/2022/Yael-Lachman/flowchart-bedtime-routine.md:4]:
```suggestion
  A((Start))-->B[Eat supper]
```

File: student-work/2022/Yael-Lachman/flowchart-bedtime-routine.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  C--No-->D{Do they have homework?}
+  D--Yes-->E[Do homework]
+  D--No-->F[Have a bath]
+  E-->F
+  F-->G{Are they clean?}
+  G--Yes-->H[Pajamas]
+  G--No-->F
+  H-->I{Do they have clothes for tomorrow?}
+  I--Yes-->J[Lay out tomorrow's clothes]
+  I--No-->K[Put clothes in washing machine]
```
Comment (Laura-Novich-OBW) [inline student-work/2022/Yael-Lachman/flowchart-bedtime-routine.md:16]:
```suggestion
  I--No-->K[Put clothes in washing machine]
  K-->J
```

File: student-work/2022/Yael-Lachman/flowchart-bedtime-routine.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  J-->L[Brush teeth]
+  K-->L
```
Comment (Laura-Novich-OBW) [inline student-work/2022/Yael-Lachman/flowchart-bedtime-routine.md:18]:
```suggestion

```

**Review by Laura-Novich-OBW — CHANGES_REQUESTED** [review]
consider the changes I made

File: student-work/2022/Yael-Lachman/flowchart-bedtime-routine.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  L-->M[Shema]
+  M-->N(End)
```
Comment (Laura-Novich-OBW) [inline student-work/2022/Yael-Lachman/flowchart-bedtime-routine.md:20]:
```suggestion
  M-->N((End))
```

**Comment by Yael0508** [comment]
Committed all your changes, thanks

**Pull request merged**